lab-traceyr/lib/cowsay.js

```diff
@@ -0,0 +1,7 @@
+var Animal = function(){};
+
+Animal.prototype.cowsay = function(string) {
+  return 'The cow says "' + string + '"';
+};
+
+exports.Animal = new Animal();
```

Comment:
This totally works, but is a bit of an anti-pattern. You probably want to go with the module.exports pattern that Tyler showed in class yesterday.